en/reference/schema-reference.html

```diff
@@ -1005,9 +1008,10 @@ <h2 id="field">field</h2>
             separately. Note that only URLs can be indexed this way, not other URIs.
             The different components are as defined by the HTTP standard:
             Scheme, hostname, port, path, query and fragment. Example:
-          <pre>
-http://mysite.mydomain.com:8080/path/shop?d=hab&amp;id=1804905709&amp;cat=100#frag1
-</pre>
+            <pre>
+              http://mysite.mydomain.com:8080/path/shop?d=hab&amp;id=1804905709&amp;cat=100#frag1
+            </pre>
```

Comment:
As the pre elements preserve whitespace, they look better without this indentation

```suggestion
<pre>
http://mysite.mydomain.com:8080/path/shop?d=hab&amp;id=1804905709&amp;cat=100#frag1
</pre>
```